app/models/slack_notifier.rb

```diff
@@ -0,0 +1,16 @@
+class SlackNotifier
+  attr_reader :client
+
+  # 環境SLACK_WEBHOOK_URLにwebhook urlを格納
+  WEBHOOK_URL = "https://hooks.slack.com/services/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"
+  CHANNEL = "#health_check"
+  USER_NAME = "Health_check_Reminder"
+
+  def initialize
+    @client = Slack::Notifier.new(WEBHOOK_URL, channel: CHANNEL, username: USER_NAME)
+  end
+
+  def send(message)
+    Slack::Notifier.new(WEBHOOK_URL, channel: CHANNEL, username: USER_NAME).ping(message)
```

Comment:
![image](https://user-images.githubusercontent.com/102507273/187598378-7e72ce46-50a7-4f69-b1e0-f991d38e710a.png)

Reply:
Slack::Notifier::APIError in SetUpsController#create
The slack API returned an error: invalid_token (HTTP Code 403)
Check the "Handling Errors" section on https://api.slack.com/incoming-webhooks for more information